### PR TITLE
add Indonesian translation

### PR DESCRIPTION
**Pull request by dreamyukii**
this pr will add Indonesian translation to timeswitch

### CODE REVIEW

**Comment by fsobolev** [comment]
Two new strings was added, please update translation

File: po/id.po [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+msgstr "Berhenti"
+
+#: src/window.py:351
+msgid "You can close the window, the timer will work in the background."
+msgstr "Anda bisa menutup jendela, pewaktu akan berjalan di latar belakang"
+
+# Translators: make sure to have a space in the end of the line.
+#: src/window.py:399
+msgid "They will be executed outside of flatpak sandbox. "
+msgstr "Ini akan dijalankan di luar isolasi flatpak."
```
Comment (fsobolev) [inline po/id.po]: Please add space at the end of the line

Comment (dreamyukii) [reply po/id.po]: done

**Comment by fsobolev** [comment]
Thank you! 😊

**Pull request merged**